src/Support/Laravel/L5ServiceProvider.php

```diff
@@ -15,10 +15,10 @@ class L5ServiceProvider extends LaravelServiceProvider
     public function boot()
     {
         $this->publishes([
-            __DIR__.'/../../config/config.php' => config_path('twilio.php'),
+            __DIR__ . '/../../config/config.php' => config_path('twilio.php'),
         ]);
 
-        $this->mergeConfigFrom(__DIR__.'/../../config/config.php', 'twilio');
+        $this->mergeConfigFrom(__DIR__ . '/../../config/config.php', 'twilio');
```

Comment:
symfony style guide is a bit stricter than psr-2. This includes removing all spaces surrounding `.` (concatenation character)